src/manifest.ts

```diff
@@ -0,0 +1,32 @@
+export interface ExtensionManifest {
+  label: string;
+  mount: string;
+  target: string;
+  permissions: string[];
+  url: string;
+}
+
+export interface WebhookManifest {
+  name: string;
+  events: string[];
+  query: string;
+  targetUrl: string;
+  isActive: boolean;
+}
+
+export interface Manifest {
```

Comment:
`AppManifest`? If we scope `ExtensionManifest` and `WebhookManifest`, maybe this also should be very explicit

Reply:
Also - maybe not in this PR - but I'd like to have Manifest strongly validated

Reply:
Was not intended to be added here. Removing from the scope.